Joinpoint Regression in R.R

```diff
@@ -61,133 +73,158 @@ ggplot(mortdata, aes(year, asdr, color = cod)) +
+  mutate(n = r / 100000 * pop) %>%
+  # rename cod to more convenient names
+  mutate(cod = case_when(
+    cod == "Accidents" ~ "accidents",
+    cod == "Cancer" ~ "cancer",
+    cod == "Heart Disease" ~ "heart",
+    cod == "Influenza and Pneumonia" ~ "pi",
+    cod == "Stroke" ~ "stroke",
+    cod == "Tuberculosis" ~ "tb"
+  )) %>%
+  # shape to wide
+  pivot_wider(
+    names_from = "cod", values_from = c("r", "n"),
+    names_glue = "{cod}.{.value}")
 
 # Let's plot a couple real quick to make sure we have what we need
 
-plot(mortdata.w$pi.n)
-plot(mortdata.w$heart.n)
-plot(mortdata.w$tb.n)
+plot(mdw$pi.n)
+plot(mdw$heart.n)
+plot(mdw$tb.n)
 
 ### LOGISTIC JOINPOINT MODELING ----
 
-# Fitting the data, finding the joinpoints. 
+# Fitting the data, finding the joinpoints.
 
-# the 'ljr' package contains a number of built-in functions, but we will be working with only three:
+# the 'ljr' package contains a number of built-in functions, but we will be
+# working with only three:
 #   1. ljr0() : fits zero joinpoints - just a straight line!
-#   2. ljr1() : fits a single joinpoint at the most significant point of change over the time period
-#   3. ljrk() : fits as many joinpoints as you want! 
-            # WARNING: ANYTHING OVER 3 CAN TAKE A LONG TIME
-            # And obviously: be careful about overfitting your model!
+#   2. ljr1() : fits a single joinpoint at the most significant point of
+#               change over the time period
+#   3. ljrk() : fits as many joinpoints as you want!
+# WARNING: ANYTHING OVER 3 CAN TAKE A LONG TIME
+# And obviously: be careful about overfitting your model!
 
-# Let's run a few of these models to see what the outputs look like. 
-# One of the great things about these models is that the output is very short and very intuitive. They also print automatically!
+# Let's run a few of these models to see what the outputs look like.
+# One of the great things about these models is that the output is
+# very short and very intuitive. They also print automatically!
 
-mdw <- mortdata.w
-rm(mortdata.w)
 
 # We'll use P&I first
-fit.1 <- ljr0(mdw$pi.n, mdw$pop, mdw$year) 
-        # arguments: number of deaths, population size (denominator of mortality rate), time
-        # the output of this fit is super straightforward: we have a y-intercept of 46.75... and a slope of -0.027...
-        # you can index these values easily as well
-                      fit.1$Coef
-                      
-fit.2 <- ljr1(mdw$pi.n, mdw$pop, mdw$year, summ = T)
-        # arguments: number of deaths, population size, time, automatically print summary table
-        # now, we have a y-int of -15.61... and some other things!
-        # g0 is the slope of the line before any joinpoints are identified. 
-        # g1 max(t-tau1,0) is the slope of the line after the joinpoint is identified. 
-        # we also have our first tau (joinpoint) estimate: 1918!
-
+# arguments: number of deaths, population size
```

Comment:
Changing comment structure so that it precedes code. Writing out `TRUE` for logicals as part of style guideline.